inc/class-saml.php

```diff
@@ -782,6 +788,9 @@ public function endLogin( $msg ) {
 	 * @return false|\WP_User
 	 */
 	public function matchUser( $net_id ) {
+		if ( ! $net_id ) {
```

Comment:
Nothing to search in wp_meta with a $net_id === false. It should not happen with the current changes, but just in case.